lib/telegram/bot/types/chat.rb

```diff
@@ -2,6 +2,8 @@ module Telegram
   module Bot
     module Types
       class Chat < Base
+        include Virtus.model(finalize: false)
```

Comment:
I am not sure it is okay to include `Virtus.model` twice.
It works but it looks dirty. I guess we can extract `Base#to_compact_hash` to mixin and include it where needed.